presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -61,6 +61,7 @@
 import org.apache.hadoop.fs.BufferedFSInputStream;
```

Comment:
can we have a (micro) benchmark for this?

Reply:
From Slack:
> That PR brought us from > 1 GB/s object read rate to under 10 MB/s for the same query

Reply:
nice! Do you know if it also increased # of requests?